hack/data-plane.sh

```diff
@@ -145,28 +145,20 @@ function k8s() {
 }
 
 function data_plane_unit_tests() {
-
-  java_ut || java_ut || fail_test "Data plane unit tests failed"
+  pushd ${DATA_PLANE_DIR} || fail_test
+  mvn clean verify --no-transfer-progress
```

Comment:
I think we need to retry here as we did before to mitigate connection reset errors